templates/scenarios/common/init-debug-vs-bot/teamsapp.local.yml

```diff
@@ -9,12 +9,12 @@ provision:
     writeToEnvironmentFile: # Write the information of installed dependencies into environment file for the specified environment variable(s).
       teamsAppId: TEAMS_APP_ID
 
-  - uses: botAadApp/create # Creates a new AAD app for bot if BOT_ID environment variable is empty
+  - uses: botAadApp/create # Creates a new or reuses an existing Azure Active Directory application for bot.
     with:
-      name: bot
-    # Output: following environment variable will be persisted in current environment's .env file.
-    # BOT_ID: the AAD app client id created for bot
-    # SECRET_BOT_PASSWORD: the AAD app client secret created for bot
+      name: ${{APP_NAME}}-${{TEAMSFX_ENV}} # The Azure Active Directory application's display name
+    writeToEnvironmentFile:
```

Comment:
@sffamily Hi Zhidi, can you help take a look at this PR in terms of wording? You can just check this file because the rest are the same.